src/avizo_service.vala

```diff
@@ -204,29 +220,49 @@ public class AvizoWindow : Gtk.Window
 			               block_height);
 		}
 
-		Gdk.cairo_set_source_rgba(ctx, bar_fg_color);
+        if(!disable_progress)
+        {
+		    Gdk.cairo_set_source_rgba(ctx, bar_fg_color);
+
+		    if (block_spacing > 0)
+		    {
+		    	for (int i = 0; i < (int) (block_count * progress); i++)
+		    	{
+		    		draw_rect(ctx, blocks_x + (block_width + block_spacing) * i,
+		    					blocks_y,
+		    					block_width,
+		    					block_height);
+		    	}
+		    }
+		    else {
+		    	var width = block_width * block_count * progress;
+		    	var height = block_height;
+		    	draw_rect(ctx, blocks_x,
+		    				blocks_y,
+		    				width,
+		    				height);
+		    }
+		}
+
 
-		if (block_spacing > 0)
+		// Draw text
+		if(disable_progress && text.length > 0)
 		{
-			for (int i = 0; i < (int) (block_count * progress); i++)
-			{
-				draw_rect(ctx, blocks_x + (block_width + block_spacing) * i,
-							blocks_y,
-							block_width,
-							block_height);
-			}
-		}
-		else {
-			var width = block_width * block_count * progress;
-			var height = block_height;
-			draw_rect(ctx, blocks_x,
-						blocks_y,
-						width,
-						height);
-		}
+            ctx.select_font_face ("Sans", Cairo.FontSlant.NORMAL, Cairo.FontWeight.BOLD);
```

Comment:
Would it make sense to be able to choose the font as well? For the font size you added that option, too.